examples/ollama/Aspire.CommunityToolkit.Hosting.Ollama.Web.MEAI/Program.cs

```diff
@@ -0,0 +1,52 @@
+using Aspire.CommunityToolkit.Hosting.Ollama.Web.MEAI.Components;
+using Microsoft.Extensions.AI;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add service defaults & Aspire components.
+builder.AddServiceDefaults();
+
+// Add services to the container.
+builder.Services.AddRazorComponents()
+    .AddInteractiveServerComponents();
+
+builder.Services.AddOutputCache();
+
+builder.AddOllamaApiClient("ollama");
+
+builder.Services.AddChatClient(
+    builder => 
+        {
+            return new OllamaChatClient(new Uri("http://localhost:11434/"), modelId: "phi3.5");
```

Comment:
This should just rely on service discovery and use the model info from the config values exposed by the App Host

Reply:
Why isn't this using 

https://github.com/CommunityToolkit/Aspire/blob/bd5e72f81db9d6ec83d9ba9ccfe99a1bb02927e8/src/Aspire.CommunityToolkit.OllamaSharp/AspireOllamaSharpExtensions.cs#L12-L19

?

Reply:
The `OllamaChatClient` is from MEAI.Ollama whereas the `IOllamaApiClient` type is what is registered by OllamaSharp.

Reply:
I'm not 100% following. We are able to use both the Aspire.OpenAI client integration and Microsoft.Extensions.AI.OpenAI together. Why can't we do it with Ollama?

See

https://github.com/dotnet/eShop/blob/1acac04d4ad4247b6661103676718850e580c40b/src/WebApp/Extensions/Extensions.cs#L117-L122

Reply:
The `AddChatClient` method provided by MEAI expects an instance of `IChatClient`, which is what we get from https://github.com/dotnet/extensions/blob/ef606f2e1f40c69d4289d15e37c05e07dc20fd14/src/Libraries/Microsoft.Extensions.AI.Ollama/OllamaChatClient.cs#L21

Presently though, the OllamaSharp library doesn't implement that interface https://github.com/awaescher/OllamaSharp/blob/c9be5f578f598ec2f0b00e68c68f3be69acdace4/src/OllamaApiClient.cs#L22

This means that we can't put it in as an alternative for MEAI to access Ollama.

Reply:
I get it now. `Microsoft.Extensions.AI.Ollama` doesn't take a dependency on OllamaSharp. It makes its own client. Unlike the M.E.AI.OpenAI library, which does depend on the OpenAI package.

Any timeline for OllamaSharp getting an implementation for `IChatClient`?

Reply:
Ah yes, sorry I should have opened with that.

I believe @SteveSandersonMS has approached the author of OllamaSharp on that topic